.github/issue_template.md

```diff
@@ -0,0 +1,19 @@
+<!--- Provide a general summary of the issue in the Title above -->
+<!--- If you have a question along the lines of "How do I do this Bash command in xonsh"
+please first look over the Bash to Xonsh translation guide: http://xon.sh/bash_to_xsh.html
+If you don't find an answer there, please do open an issue! -->
+
+## xonfig
+<!--- Please post the output of the `xonfig` command (run from inside xonsh) so we know more about your current setup -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
+<!--- If part of your bug report is a traceback, please first enter debug mode before triggering the error
+To enter debug mode, set the environment variable `XONSH_DEBUG=1` _before_ starting `xonsh`.  
+On Linux and OSX, an easy way to to do this is to run `env XONSH_DEBUG=1 xonsh` -->
```

Comment:
On Windows... cmd: `set XONSH_DEBUG=1&&xonsh` and Powershell: `$Env:XONSH_DEBUG=1;xonsh`